background/networks.ts

```diff
@@ -9,8 +9,9 @@ import type {
   EnrichedEVMTransactionSignatureRequest,
   PartialTransactionRequestWithFrom,
 } from "./services/enrichment"
+import { ChainData } from "./constants"
 
-export const CURRENT_QUAI_CHAIN_ID = "1337"
+export const CURRENT_QUAI_CHAIN_ID = "9000" // TODO: Remove this, should be dynamic
```

Comment:
Please create an issue for this TODO

Reply:
I deleted CURRENT_QUAI_CHAIN_ID as it is unused